src/components/Filters.js

```diff
@@ -194,7 +194,7 @@ export default class Filters {
    * Restore state from the URL hash
    */
   restoreState() {
-    const params = document.location.href.split('!mt-filters=');
+    const params = document.location.href.replace(/%21mt/g, '!mt').split('!mt-filters=');
```

Comment:
the new ``replace()`` looks good, but is the following ``split()`` on ``'!mt-filters='`` still work given that it's now ``%21mt``?

Reply:
We actually replacing all `%21mt` in the URL with `!mt`, and then extract its content.